zentral/contrib/munki/templates/munki/scriptcheck_detail.html

```diff
@@ -39,7 +39,7 @@ <h3 class="m-0 fs-5 text-secondary">Script check</h3>
     </div>
 
     <div class="table-responsive mb-3">
-        <table class="table table-light table-striped align-middle table-object-properties">
+        <table class="table table-light table-striped align-middle table-object-properties table-fixed-scrolleable">
```

Comment:
Is it necessary to make the whole table scrollable when only one cell can overflow and this cell is also scrollable?

Reply:
The whole table scrollable? That is being achieved with `table-responsive`.
Do you know what the property `table-layout: fixed;` does and how it plays with `pre` elements inside a table cell?